test_verified_group_vs_delete_server_after is flaky

Failures look like this:
```
        lp.sec("ac2: sending message re-gossiping Autocrypt keys")
        chat2.send_text("hi2")

        lp.sec("ac2_offl: receiving message")
        ev = ac2_offl._evtracker.get_matching("DC_EVENT_INCOMING_MSG|DC_EVENT_MSGS_CHANGED")
        msg_in = ac2_offl.get_message_by_id(ev.data2)
>       assert msg_in.is_system_message()
E       AssertionError: assert False
E        +  where False = is_system_message()
E        +    where is_system_message = <Message outgoing sys=False 'hi2' id=14 sender=1/ci-dx3spq@nine.testrun.org chat=12/hello>.is_system_message
```

Attached is a recent failure from CI: [job-logs.txt](https://github.com/user-attachments/files/17382539/job-logs.txt)

--- a/src/mimefactory.rs
+++ b/src/mimefactory.rs
@@ -352,7 +352,11 @@ impl MimeFactory {
                     // beside key- and member-changes, force a periodic re-gossip.
                     let gossiped_timestamp = chat.id.get_gossiped_timestamp(context).await?;
                     let gossip_period = context.get_config_i64(Config::GossipPeriod).await?;
-                    if time() >= gossiped_timestamp + gossip_period {
+                    // `gossip_period == 0` is a special case for testing,
+                    // enabling gossip in every message.
+                    // Othewise "smeared timestamps" may result in the condition
+                    // to fail even if the clock is monotonic.
+                    if gossip_period == 0 || time() >= gossiped_timestamp + gossip_period {
                         Ok(true)
                     } else {
                         Ok(false)

